extensions/arc/src/common/date.ts

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+// This file was copied from src/vs/base/common/date.ts
+const minute = 60;
+const hour = minute * 60;
+const day = hour * 24;
+const week = day * 7;
+const month = day * 30;
+const year = day * 365;
+
+export function fromNow(date: number | Date, appendAgoLabel?: boolean): string {
+	if (typeof date !== 'number') {
+		date = date.getTime();
+	}
+
+	const seconds = Math.round((new Date().getTime() - date) / 1000);
+	if (seconds < -30) {
+		return localize('date.fromNow.in', 'in {0}', fromNow(new Date().getTime() + seconds * 1000, false));
```

Comment:
@Charles-Gagnon I figure this was too easy.  I have to add localized constants for each of these, right?

Reply:
I think in this case it's fine to leave it here in case the original is updated and we want to update this version. Less work that way. The localized constants is just a slight convenience thing for checking what localized strings there are but it's nothing critical.